layouts/shortcodes/docs/instrumentation/index-adopters.md

```diff
@@ -0,0 +1,18 @@
+<!-- prettier-ignore -->
+{{ $lang := .Get 0 -}}
+{{ $Lang := $lang | humanize -}}
+{{ $howMany := .Get 1 | default 10 -}}
```

Comment:
Btw, @svrnm, I don't know if you noticed, but I put a cap on the number of adopters. There's a comment at the end that says "for the complete list ... see". This will avoid excessively long lists.

Reply:
I like it!